### ISSUE TITLE
Reverse Proxy rewriting with comma?

### ISSUE THREAD

**Issue by jamieshaw** [morpheus65535/bazarr#232]
I'm having some issues setting up a properly configured reverse proxy under Apache – I followed the documentation on the Wiki to the letter, but I'm experiencing strange behaviour where the domain is rewritten with a comma…

So, for example:
`https://subdomain.domain.com/bazarr`
is rewritten to
`https://subdomain.domain.com%2C%20subdomain.domain.com/bazarr`
`(https://subdomain.domain.com, subdomain.domain.com/bazarr)`

Any ideas why this would be? A bug in my Apache config, or something funky in Bazarr?
Don't experience this issue with any other reverse proxies currently running.

**Comment by morpheus65535** [comment]
What version of Apache are you using? Could you post your configuration? Have you tried adding slash at the end of your location, proxypass and reverseproxypass?

**Comment by jamieshaw** [comment]
Version 2.4.33 – the version supplied with macOS Server.

Configuration is a bit of a mess as it's all handed my the Server GUI which proxies it's on Apache server with another apache server, then routes everything through virtual hosts from a single domain.

But on the simple side, I can confirm that the trailing slashes are included. It'll also append `/login` to the URL when rewriting too.

**Comment by morpheus65535** [comment]
I don't have a clue how this work on macOS Server. I'll leave this open in case someone can help you with it.

**Comment by jamieshaw** [comment]
So I've just tried connecting to `https://subdomain.domain.com/bazarr/login`, which works and skips the rewrite – are there any redirects occurring when hitting `/bazarr/`?

**Comment by morpheus65535** [comment]
Yes, there is a couple of redirect: `/wizard` on first run, `/series` if Sonarr integration is enabled or `/movies` if Sonarr is disabled but Radarr integration is enabled, `/settings` if no integration are enabled, etc. Both proxy configuration have been tested under Linux and redirect do not cause issues. Maybe something special on macOS?
